extensions/amp-story-interactive/0.1/test/helpers.js

```diff
@@ -201,3 +201,22 @@ export const addThresholdsToInteractive = (interactive, thresholdList) => {
     );
   });
 };
+
+export const MOCK_URL = 'https://amp.dev';
+
+/**
+ *
+ * @param {!../../../../service/xhr-impl.Xhr} xhrMock
+ * @param {!Object} json
+ */
+export function mockXhrWithJson(xhrMock, json) {
```

Comment:
I'm really not a fan of these very simple helpers, they make it harder for people to understand/refactor/add tests. I think it's OK to remove it and repeat the code in the tests.
If you need to change the `json` a lot, you can mock only once in a `before` and each test can update the `json` "global" variable that the mock returns.

Reply:
I saw https://github.com/ampproject/amphtml/blob/main/extensions/amp-call-tracking/0.1/test/test-amp-call-tracking.js#L50 and thought it made sense to extract a function like that. I worry if I copy paste in all the tests the setup calls, 50% of the test logic is just mocking a JSON response.

Mocking it on the `before` sg, since we can avoid all that extra verbosity.